utils.ts

```diff
@@ -24,6 +24,20 @@ export type ServerlessResponse = {
   statusCode: number
 }
 
+/**
+ * This method removes all characters Pelias doesn't support.
+ * Unfortunately, these characters not only don't match if they're found in the
+ * elasticsearch record, but make the query fail and return no results.
+ * Therefore, they are removed using this method. The search still completes
+ * as one would expect: "ab @ c" gets converted to "ab  c" which still matches
+ * an item named "ab @ c"
+ * @param queryString The *URL decoded* string with invalid characters to be cleaned
+ * @returns           The string with invalid characters replaced
+ */
+export const makeQueryPeliasCompatible = (queryString: string): string => {
+  return queryString.replace('@', ' ').replace('&', ' ')
```

Comment:
This will replace [**up to one** instance](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/replace) of `@` and `&`, so either say so in the comments if we only expect one such characters, or change the function call so it replaces all instances.